components/Form/index.js

```diff
@@ -37,68 +26,68 @@ export default function Form() {
   const handleSubmit = (event) => {
     event.preventDefault();
 
-    const formData = new FormData(event.target);
-    const data = Object.fromEntries(formData);
-
-    //neues Projekt-Objekt erstellen
+    //create new project-object
     const project = {
       name: event.target.projectname.value,
       items: items,
+      id: uuidv4(),
     };
 
     const resetItems = partList.map((item) => {
       return { ...item, price: 0, value: "" };
     });
 
-    //das neue Projekt zur Liste der Produkte hinzufügen
-    setProjects([...projects, project]);
+    //add new project to list
+    addProject(project);
 
-    // Formular-Eingaben zurücksetzen
+    // reset form
     setItems(resetItems);
 
     setProjectName("");
     event.target.reset();
   };
 
   return (
-    <form onSubmit={handleSubmit}>
-      <input
-        type="text"
-        name="projectname"
-        placeholder="project name"
-        value={projectName}
-        onChange={handleProjectNameChange}
-      />
-      {items.map((item, index) => {
-        return (
-          <div key={item.id} className="entry-form">
-            <input
-              name={item.name}
-              type="text"
-              placeholder={item.name}
-              value={item.value}
-              onChange={(event) =>
-                handleItemChange(index, "value", event.target.value)
-              }
-            />
+    <>
+      <form onSubmit={handleSubmit}>
```

Comment:
it seems that you have a similar form in `EditForm` already. Maybe make a component out of the form? Might be a possible solution.

Reply:
...that was my idea too, but that are similiar forms they do different things...so instead of prop drilling i decide to duplicate -> duplication is cheaper than a wrong abstraction
...i will keep this in mind, if i need this form again